static/js/src/advantage/react/components/Subscriptions/SubscriptionCancel/SubscriptionCancel.test.tsx

```diff
@@ -1,14 +1,283 @@
-import { Modal } from "@canonical/react-components";
+import { ActionButton, Modal, Notification } from "@canonical/react-components";
 import React from "react";
-import { shallow } from "enzyme";
+import { mount } from "enzyme";
```

Comment:
would you consider trying out `@testing-library/react`? 🙃

Reply:
This is not the right point for that, this work is nearly complete and it would intro further fragmentation into our code.